modules/cms/widgets/MediaManager.php

```diff
@@ -985,10 +985,14 @@ protected function checkUploadPostback()
 
             $path = Input::get('path');
             $path = MediaLibrary::validatePath($path);
-
+            
+            $realpath = empty(trim($uploadedFile->getRealPath())) 
+                            ? $uploadedFile->getPath().DIRECTORY_SEPARATOR.$uploadedFile->getFileName() 
+                            : $uploadedFile->getRealPath()
+                        ;
             MediaLibrary::instance()->put(
                 $path.'/'.$fileName,
-                File::get($uploadedFile->getRealPath())
+                File::get($realpath))
```

Comment:
I think this is because of a quirk in IIS that sometimes returns null on `getRealPath()` so it needs to be stitched together manually in these cases.